server/src/tools/client_credentials_admin.py

```diff
@@ -87,6 +87,29 @@ def handle_allowed_queues_input() -> list:
     return allowed_queues
 
 
+def handle_max_reservation_input() -> list:
+    """Gets a user inputted max_reservation_time dict"""
+    max_reservation = {}
+    while True:
+        max_reservation_input = input(
+            "Do you want to specify a queue to set"
+            + " max reservation time for? (y/n): "
```

Comment:
I just noticed this, because the + blended in with the other "+"s on the left :)
But you don't really need that + to concatenate the strings. Any string within parentheses is going to get automatically concatenated. This is similar to you how can handle things like line length limits by adding ()s and breaking a string across lines. It's a minor thing but let's go ahead and clean it up since we're also waiting for the other thing to land anyway.